widgets/Dataplex-batchsource.json

```diff
@@ -0,0 +1,371 @@
+            "placeholder": "Ex: us-central1"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Lake Id",
+          "name": "lake",
+          "widget-attributes": {
+            "placeholder": "Ex: Lake"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Zone Id",
+          "name": "zone",
+          "widget-attributes": {
+            "placeholder": "Ex: zone"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Asset Id",
+          "name": "asset",
+          "widget-attributes": {
+            "placeholder": "Ex: Asset"
+          }
+        },
+        {
+          "widget-type": "radio-group",
+          "label": "Asset type",
+          "name": "assetType",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "BIGQUERY_DATASET",
+            "options": [
+              {
+                "id": "BIGQUERY_DATASET",
+                "label": "BigQuery Dataset"
+              },
+              {
+                "id": "STORAGE_BUCKET",
+                "label": "Storage Bucket"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "label": "Asset Details",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Table",
+          "name": "table",
+          "widget-attributes": {
+            "placeholder": "Table to read from"
+          },
+          "plugin-function": {
+            "label": "Get Schema",
+            "widget": "outputSchema",
+            "output-property": "schema",
+            "omit-properties": [
+              {
+                "name": "schema"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Partition Start Date",
+          "name": "partitionFrom",
+          "widget-attributes": {
+            "placeholder": "Partition start date in format yyyy-MM-dd"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Partition End Date",
+          "name": "partitionTo",
+          "widget-attributes": {
+            "placeholder": "Partition end date in format yyyy-MM-dd"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Filter",
+          "name": "filter",
+          "widget-attributes" : {
+            "placeholder": "Filters out rows that do not match the given condition"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Path",
+          "name": "path",
+          "widget-attributes": {
+            "placeholder": "gs://<bucket>/path/to/input"
+          }
+        },
+        {
+          "widget-type": "plugin-list",
+          "label": "Format",
+          "name": "format",
+          "widget-attributes": {
+            "plugin-type": "validatingInputFormat"
```

Comment:
Specify the list of supported plugins using select widget.
You had done that in your previous PR - https://github.com/data-integrations/google-cloud/pull/762/files/a1a1fbdd5d882d71aa4e679f48d616f9d5d7b3f3#diff-88a190e0c4b19643ef0a86e658179d5544a05c571aecb2f40507e724f908643cR165